oci/oci.go

```diff
@@ -275,7 +275,7 @@ func (r *Runtime) ExecSync(c *Container, command []string, timeout int64) (resp
 func (r *Runtime) StopContainer(c *Container) error {
 	c.opLock.Lock()
 	defer c.opLock.Unlock()
-	if err := utils.ExecCmdWithStdStreams(os.Stdin, os.Stdout, os.Stderr, r.path, "kill", c.name); err != nil {
+	if err := utils.ExecCmdWithStdStreams(os.Stdin, os.Stdout, os.Stderr, r.path, "kill", c.name, "KILL"); err != nil {
```

Comment:
Is this sending a `SIGKILL` to the container?  If so, it seems more "polite" to send a `SIGINT` first, to give the application time to clean up, before sending `SIGKILL` to forcibly kill the processes.  Is a forcible/uncatchable termination signal required per the CRI spec?